src/cinder/app/cocoa/AppImplMac.mm

```diff
@@ -628,9 +628,21 @@ - (void)resize
 	NSSize nsSize = [mCinderView frame].size;
 	mSize = cinder::ivec2( nsSize.width, nsSize.height );
 
+	NSRect frame = [mWin frame];
+	NSRect content = [mWin contentRectForFrameRect:frame];
+	
+	ivec2 prevPos = mPos;	
+	mPos = ivec2( content.origin.x, [[[NSScreen screens] objectAtIndex:0] frame].size.height - frame.origin.y - content.size.height );
```

Comment:
Ahh, interesting - it should be pretty easy to test and find out if it is an issue, just move the window to a secondary display and then hit the relevant method.